pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -37,81 +37,72 @@ const _ = cockpit.gettext;
  * Enables showing shell and ansible script. Shell one is mandatory and ansible one can be omitted.
  *
  */
-export class ModificationsExportDialog extends React.Component {
-    constructor(props) {
-        super(props);
+export const ModificationsExportDialog = ({ show, onClose, shell, ansible }) => {
+    const [active_tab, setActiveTab] = React.useState("shell");
+    const [copied, setCopied] = React.useState(false);
+    let timeoutId = null;
```

Comment:
This will run on every render, are you sure that this is right? It smells like it would need to go into `useInit()`?

Reply:
I'm wondering if we should keep it in state instead?

Reply:
It would certainly work. Changing it would trigger an unnecessary render though, hence the useInit() idea.